integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/HorreumResource.java

```diff
@@ -0,0 +1,52 @@
+package com.redhat.service.smartevents.integration.tests.resources;
+
+import java.time.Instant;
+
+import com.redhat.service.smartevents.integration.tests.common.Utils;
+
+import io.cucumber.java.AfterAll;
+import io.cucumber.java.BeforeAll;
+
+import software.tnb.common.service.ServiceFactory;
+import software.tnb.horreum.service.Horreum;
+
+public class HorreumResource {
+    private static final boolean HORREUM_RESULTS_UPLOAD = Boolean.valueOf(System.getProperty("performance.horreum.results.upload"));
+    private static final String HORREUM_TEST_RESULT_SCOPE = "PUBLIC";
+
+    private static String HORREUM_TEAM_NAME;
+    private static String HORREUM_TEST_SCHEMA;
+
+    public static Horreum horreum = ServiceFactory.create(Horreum.class);
```

Comment:
would be nice to update the sample file we have for the [localconfig.yaml](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/localconfig-example.yaml) file regarding TNB config, so it can be pretty straight forward to test locally